recirq/optimize/mpg_test.py

```diff
@@ -0,0 +1,101 @@
+# Copyright 2020 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+
+from recirq.optimize.mpg import model_policy_gradient
+
+
+def sum_of_squares(x):
+    return np.sum(x**2).item()
+
+
+def test_model_policy_gradient():
```

Comment:
these tests take ~45 seconds on my machine, which is too long for a unit test. Can we provide a more concise check of the functionality?

Reply:
lower the criterion